disnake/http.py

```diff
@@ -225,12 +225,12 @@ def __init__(
         self,
         connector: Optional[aiohttp.BaseConnector] = None,
         *,
+        loop: asyncio.AbstractEventLoop,
         proxy: Optional[str] = None,
         proxy_auth: Optional[aiohttp.BasicAuth] = None,
-        loop: Optional[asyncio.AbstractEventLoop] = None,
         unsync_clock: bool = True,
     ) -> None:
-        self.loop: asyncio.AbstractEventLoop = asyncio.get_event_loop() if loop is None else loop
+        self.loop: asyncio.AbstractEventLoop = loop
```

Comment:
That parameter is deprecated since aiohttp 2.0, and there is also deprecation warnings from aiohttp that you can't create a ClientSession without an already running event loop. 

We don't need to be explicit, and we shouldn't be explicit in this case either.